brewery-app/src/components/Homepage.js

```diff
@@ -1,15 +1,28 @@
-import React from 'react'
+import React, {useState} from 'react'
+import styled from 'styled-components';
 
 import SearchForm from './SearchForm';
 import MapContainer from './MapContainer';
+import ResultsList from './ResultsList';
 
+const Div = styled.div`
```

Comment:
Can you give this a more descriptive name, like HomePageContents or MainContent or something. This can be used like a class name for styling and makes the code more readable with descriptive names